src/cli/input-from-terminal.js

```diff
@@ -0,0 +1,32 @@
+"use strict";
+
+const readline = require("readline");
+const { stdin: input } = require("node:process");
```

Comment:
We should use `stdin` directly.

Reply:
Made all the changes except this one.
I do not understand this comment.

Reply:
```suggestion
const { stdin } = require("node:process");
```